profiles/local.yaml

```diff
@@ -5,27 +5,28 @@ experiment:
 
 # Set the registry server URI. This property is especially useful if you have a registry
 # server that’s different from the tracking server.
-# model_registry:
-#   uri: sqlite:////tmp/registry.db
+model_registry:
+  # registry_uri: "sqlite:////tmp/registry.db"
+  # FIXME::REQUIRED: Specifies the name of the Registered Model to use when registering a trained
+  #                 model to the MLflow Model Registry.
+  model_name: "taxi_fare_regressor"
+  # model_uri: "models/model.pkl"
 
-# Use a small sample from the TLC Trip Record Dataset for exploring the MLflow Regression Pipeline
-# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
-INGEST_DATA_LOCATION: "./data/sample.parquet"
-# Use a larger section of the TLC Trip Record Dataset for training higher-fidelity models
-# INGEST_DATA_LOCATION: "https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2022-01.parquet"
-# Specify the format of the dataset
-INGEST_DATA_FORMAT: parquet
-# Use a small sample from the TLC Trip Record Dataset for exploring the batch scoring feature of the
-# MLflow Regression Pipeline
-# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
-INGEST_SCORING_DATA_LOCATION: "./data/sample.parquet"
-# Use a larger section of the TLC Trip Record Dataset for the batch scoring feature
-# INGEST_SCORING_DATA_LOCATION: "https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2022-01.parquet"
-# Specify the format of the dataset
-INGEST_SCORING_DATA_FORMAT: parquet
 # Override the default train / validation / test dataset split ratios
 SPLIT_RATIOS: [0.80, 0.10, 0.10]
-# Specify the output location of the batch scoring predict step
-SCORED_OUTPUT_DATA_LOCATION: "./data/sample_output.parquet"
-# Specify the output format of the batch scoring predict step
-SCORED_OUTPUT_DATA_FORMAT: parquet
\ No newline at end of file
+
+INGEST_STEP_CONFIG:
+    # For different options please read: https://github.com/mlflow/mlp-regression-template#ingest-step
+    using: parquet
+    location: "./data/sample.parquet"
+
+REGISTER_STEP_CONFIG:
+  # For different options please read: https://github.com/mlflow/mlp-regression-template#batch-scoring
+  using: parquet
+  allow_non_validated_model: false
+  location: "./data/sample.parquet"
+
+PREDICT_STEP_CONFIG:
+  # For different options please read: https://github.com/mlflow/mlp-regression-template#predict-step
+  using: parquet
```

Comment:
What is the `using` parameter? Shouldn't this be `output_format` instead?

Reply:
We would like to move to this new `using` parameter to make it more consistent with other steps.

Reply:
* Shall we rename "predict" to "batch_score"? This matches "ingest_scoring" and "scored_data". Adding "batch_" avoids the confusion with sklearn's score(). 

* I feel `using: parquet_reader` and `using: parquet_writer` is more accurate but redundant. And here we only need to describe the output configs of `batch_score` instead of the whole `batch_score`. So how about:

~~~
batch_score:
  output:
    using: parquet
    location: ...
~~~

In the future we might allow custom writers, the config might change to:

~~~
batch_score:
  output:
    using: custom
    writer_fn: my_writer
    location: ...
~~~

Reply:
Sure we can rename the predict step to `batch_score` everywhere if that makes more sense. I could do that in documentation, pipeline_dag and code as well. What do others things before i move ahead with that suggestion? 

@mengxr other than that, is there anything else we need to change? Do all the other changes make sense? 

cc @jinzhang21 @jerrylian-db

Reply:
Other changes LGTM

Reply:
I like @mengxr 's suggestion on adding `output` before `using`.

On renaming `predict`, I think it's a much bigger change we should deliberate internally a bit more before committing to it. Let's not do it in this PR.

Reply:
SGTM